# cm2.c: error: implicit declaration of function 'asprintf' is invalid in C99

--- src/cm2.c.orig	2022-07-20 16:56:48.000000000 +0700
+++ src/cm2.c	2022-12-29 04:49:30.000000000 +0700
@@ -19,8 +19,12 @@
  */
 
 /*{{{ includes, etc. */
+#ifdef __APPLE__
+#define _DARWIN_C_SOURCE
+#else
 #define _POSIX_C_SOURCE 200112L
 #define _GNU_SOURCE
+#endif
 #include "config.h"
 #include <stdio.h>
 #include <stdlib.h>
